image/default_image.go

```diff
@@ -135,7 +135,13 @@ func (d DefaultImageService) Delete(imageName string) error {
 
 	image, err = imageutils.GetImage(named.Raw())
```

Comment:
sealer images get images from images_metadata.json first, I guess we can check image existence from imageMetadataMap